### PR TITLE
Porting renderers and emissive blockers from Bay.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
WIP, DNM, proper PR body with credit to PRs to come.

### CODE REVIEW

File: code/game/atoms_init.dm [hunk 1]
```diff
@@ -107,6 +108,13 @@
 	if (!ml && loc)
 		loc.Entered(src, null)
 
+	// Handle emissive blockers here to avoid an override in atoms_movable.dm.
+	var/emissive_block = update_emissive_blocker()
+	if(emissive_block)
+		overlays += emissive_block
```
Comment (MistakeNot4892) [inline code/game/atoms_init.dm]: I think this probably needs to be a priority overlay instead?
@Lohikar would appreciate your input into the `managed_overlays` stuff.

Comment (Lohikar) [reply code/game/atoms_init.dm]: Hard no on managed overlays. This should be done with priority overlays.

Comment (Lohikar) [reply code/game/atoms_init.dm]: This is also going to require every movable in the game be mangled, mind.

**Comment by MistakeNot4892** [comment]
This is beyond me currently, will reopen someday down the track if I manage to get my head around render targets and filters.

**Pull request closed without merging**